server/jest-setup.js

```diff
@@ -4,12 +4,14 @@ const replaySetup = require('../test/helpers/replay-setup')
 replaySetup('success')
 
 jest.mock('@pubsweet/component-send-email', () => ({
+  _sendPromise: null,
   mails: [],
   send(mailData) {
-    return new Promise((resolve, reject) => {
+    this._sendPromise = new Promise((resolve, reject) => {
```

Comment:
This allows us to await the Promise so we don't have to [wait with a loop](https://github.com/elifesciences/elife-xpub/blob/1496-send-email/server/xpub-server/entities/manuscript/resolvers/submitManuscript.test.js). This will be refactored to use the new way in a separate PR.